scripts/data_transfer/copy_chr22_skat_input.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -m mv gs://cpg-tob-wgs-main/CellRegMap_input_files/plink_files/plink_chr22.bed gs://cpg-tob-wgs-test/v0/skat/
```

Comment:
AFAIK the usage of -m is different, the syntax would be:

```
gsutil -m mv File File File Target
```

here each file is referenced in a different command, so they can't be copied in parallel

Reply:
i tried to address this, please check?